specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2022-05-01/machineLearningServices.json

```diff
@@ -3975,7 +3975,11 @@
             "Disabled",
             "Enabled",
             "Auto"
-          ]
+          ],
+          "x-ms-enum": {
+            "name": "status",
```

Comment:
most other x-ms-enum types in here are PascalCased - consider making this `"Status"` instead of `"status"`

Reply:
fixed